src/_functions.scss

```diff
@@ -1,16 +1,15 @@
-/// Calculate the columns and offsets width based in the column count, the
+/// Calculates the columns/offsets width based on the column count, the
```

Comment:
If we're scrutinizing grammar, technically "columns/offsets" is possessive of "width" and should have apostrophes to indicate that. However, I recommend rephrasing it to say "Calculates the width of columns/offsets" instead.